core/src/components/card/card.tsx

```diff
@@ -108,7 +109,7 @@ export class SddsCard {
             </div>
           </div>
         )}
-        {this.bodyDivider && <div class={`sdds-divider`}></div>}
+        {this.bodyDivider && <div class={`tds-divider`}></div>}
```

Comment:
Could this use our own divider component instead? :)

Reply:
Not only that we could use it, but probably we need to use it as classes are no longer available. Great catch! I will inject the component, but most probably it won't be visible until the divider component is updated with a new prefix.